libraries/helper.rb

```diff
@@ -1,5 +1,11 @@
 module Line
   module Helper
+    def embedded_eol(line)
```

Comment:
maybe name this `embedded_eol_guard()` or `validate_no_eol()`? I feel like a function `embedded_eol(string) -> string` would add an EOL to the end of the string, or otherwise /add/ EOL to the string, which obviously isn't what's happening.